dae-alright/src/components/Header/HeaderWraper/index.js

```diff
@@ -0,0 +1,17 @@
+import React from "react"
+import "./HeaderWraper.scss"
+
+function HeaderWraper(props) {
+    return (
+        <header className="header-wrapper">
+          {props.renderProp}
```

Comment:
Hey, you doing this wrong again. You are supposed to call the function.

Reply:
Resolved